tasks/eth/1-security-council-phase-0/README.md

```diff
@@ -3,19 +3,19 @@
 Status: DRAFT, NOT READY TO SIGN
 
 > [!IMPORTANT] !!! DO NOT SIGN using this playbook yet, as the
-> Security Council membership ratification proposal is not passed, and
-> the final multisig is not set up yet.
+> official security council addresses are not published yet.
 
 ## Objective
 
 This is the playbook for executing the Security Council Phase 0 as
-approved by Governance. There are two governance proposals related to
-this:
+approved by Governance. There are two governance proposals and one
+forum update related to this:
 
 1. [Security Council: Vote #1](https://vote.optimism.io/proposals/27439950952007920118525230291344523079212068327713298769307857575418374325849).
 2. [Security Council Membership Ratification](https://vote.optimism.io/proposals/85591583404433237270543189567126336043697987369929953414380041066767718361144).
+3. [Governance Post containing the threshold and signer addresses](https://gov.optimism.io/t/security-council-vote-2-initial-member-ratification/7118/19)
```

Comment:
nit, changes the link to go to a comment from "system" confirming that 4/14 is expected threshold for now
```suggestion
3. [Governance Post containing the threshold and signer addresses](https://gov.optimism.io/t/security-council-vote-2-initial-member-ratification/7118/13)
```